cmd/server/types.go

```diff
@@ -13,6 +13,12 @@ type androidVariant struct {
 	variant
 }
 
+type iOSVariant struct {
+	Cert []byte `json:"cert"`
```

Comment:
I think we need a name here too

Reply:
doesn't the name come from the abstract Variant struct?

Reply:
i think the name is included in the other struct, its getting included in the POST anyways 

```"pass_phrase":"redhat","name":"myapp","description":"","variantID":"424c7b39-e25c-436b-8d49-754226a2dbe8","secret":"f86bd4ac-782a-408a-983f-6bc74ac8e318"}}```

Reply:
@cfoskin see my CURL

`passPhrase` - not `pass_phrase` etc

Reply:
@matzew  Yep thanks! Goland auto-generate!

Reply:
the `production: bool` flag is also missing in the payload

The model is described here:
https://aerogear.org/docs/specs/aerogear-unifiedpush-rest/index.html#-1361260775